geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaDataStore.scala

```diff
@@ -69,6 +70,14 @@ class KafkaDataStore(override val zookeepers: String,
     super[ContentDataStore].dispose()
     super[KafkaDataStoreSchemaManager].dispose()
   }
+
+  def getKafkaSchema(typeName: String): SimpleFeatureType = getSchema(new NameImpl(typeName))
```

Comment:
you shouldn't need this method at all - there is already a getSchema(String) method in ContentDataStore that you can call - it delegates to getFeatureSource.getSchema, but no point in re-implementing it.